react-app/src/components/NavBar.js

```diff
@@ -40,6 +40,12 @@ const StyledNavElement = styled.header`
 
   .search-wrapper {
     border-radius: 30px;
+<<<<<<< HEAD
```

Comment:
Missed some merge markers when doing a merge. Looks like it was a bit of styling you had done to the navbar but that should all be resolved by #54 and should be able to just remove any of the changes you made.